Xamarin.Forms.Platform.UAP/Platform.cs

```diff
@@ -297,16 +299,16 @@ async void SetCurrent(Page newPage, bool popping = false, Action completedCallba
 					return;
 
 #pragma warning disable CS0618 // Type or member is obsolete
-			// The Platform property is no longer necessary, but we have to set it because some third-party
-			// library might still be retrieving it and using it
-			newPage.Platform = this;
+				// The Platform property is no longer necessary, but we have to set it because some third-party
+				// library might still be retrieving it and using it
+				newPage.Platform = this;
 #pragma warning restore CS0618 // Type or member is obsolete
 
-			if (_currentPage != null)
-			{
-				Page previousPage = _currentPage;
-				IVisualElementRenderer previousRenderer = GetRenderer(previousPage);
-				_container.Children.Remove(previousRenderer.ContainerElement);
+				if (_currentPage != null)
```

Comment:
are these style changes needed ?

Reply:
there was invalid indention (3 tabs replaced with 4 tabs). Now all items in the Try block